plugins/modules/vyos_command.py

```diff
@@ -43,12 +43,16 @@
       argument is provided, the module is not returned until the condition is satisfied
```

Comment:
Adding a quick note about the variation of elements in this list may be worth it. For e.g., https://github.com/ansible-collections/cisco.nxos/pull/126/files#diff-8edd2850a1c49131fc5d71280acd74bbR32 (copied from ios_config)